src/app/tests/suites/certification/Test_TC_OPCREDS_3_7.yaml

```diff
@@ -11,200 +11,252 @@
-          [1658223679.580915][6136:6141] CHIP:TOO:     attestationSignature: EB731B40B20501AF32C468AA522948F7848D3AEDFA24D9A879575B4A265886C97109EE0DE1ECEB969B1A7F98F127DB4C275292B986BF8DA56EF7B16DA8EC8ABE
-          [1658223679.580943][6136:6141] CHIP:TOO:    }
-      disabled: true
+      identity: "alpha"
+      command: "CSRRequest"
+      cluster: "Operational Credentials"
+      arguments:
+          values:
+              - name: CSRNonce
+                value: "\x00\x01\x02\x03\x04\x05\x06\x07\x00\x01\x02\x03\x04\x05\x06\x07\x00\x01\x02\x03\x04\x05\x06\x07\x00\x01\x02\x03\x04\x05\x06\x07"
+      response:
+          values:
+              - name: "NOCSRElements"
+                saveAs: NOCSRElements
+              - name: "AttestationSignature"
+                saveAs: attestationSignature
+      # verification: "Step 5: Verify that the DUT responds with the CSRResponse command."
+
+    - label: "Step 6.1: Read the commissioner root certificate from TH2's fabric. Save RCAC as Root_CA_Certificate_TH2"
+      # PICS:
+      identity: "beta"
+      cluster: "CommissionerCommands"
+      command: "GetCommissionerRootCertificate"
+      response:
+          values:
+              - name: "RCAC"
+                saveAs: Root_CA_Certificate_TH2
+      # verification: ""
 
     - label:
-          "Step 7: TH2 generates the NOC, the Root CA Certificate and ICAC using
-          the CSR elements from Step 5 and selects an IPK, all for use by TH2.
-          The certificates shall have their subjects padded with additional data
-          such that they are each the maximum certificate size of 400 bytes when
-          encoded in the MatterCertificateEncoding. Save RCAC as
-          Root_CA_Certificate_TH2 Save ICAC as Intermediate_Certificate_TH2 Save
-          NOC as Node_Operational_Certificate_TH2 Save IPK as IPK_TH2 Extract
-          the RCAC public key and save as Root_Public_Key_TH2"
-      verification: |
-
-      disabled: true
-
-    - label:
-          "Step 8: TH1 sends AddTrustedRootCertificate command to DUT with
-          RootCACertificate set to Root_CA_Certificate_TH2"
+          "Step 6.2: TH2 generates the NOC, the Root CA Certificate and ICAC using the CSR elements from Step 5 and selects an IPK, all for use by TH2. The certificates shall have their subjects padded with additional data such that they are each the maximum certificate size of 400 bytes when encoded in the MatterCertificateEncoding.
+          Save ICAC as Intermediate_Certificate_TH2.
+          Save NOC as Node_Operational_Certificate_TH2.
+          Save IPK as IPK_TH2.
+          Extract the RCAC public key and save as Root_Public_Key_TH2."
+      # PICS:
+      identity: "beta"
+      cluster: "CommissionerCommands"
+      command: "IssueNocChain"
```

Comment:
Is this actually doing the "padded with additional data" bit?  If not, please file a followup issue to do that.

Reply:
Doesn't appear to be, either way the '400 bytes' max is not needed here, will submit a test plan issue to address this

Reply:
Addressed in commit 851d536a95d0d1232bca39f21086ba1ae215cc90